fabric-rendering-v1/src/main/java/net/fabricmc/fabric/mixin/client/rendering/MixinWorldRenderer.java

```diff
@@ -41,17 +41,18 @@
 import net.minecraft.util.math.BlockPos;
 import net.minecraft.util.math.Matrix4f;
 
+import net.fabricmc.fabric.api.client.rendering.v1.EnvironmentRenderers;
 import net.fabricmc.fabric.api.client.rendering.v1.InvalidateRenderStateCallback;
 import net.fabricmc.fabric.api.client.rendering.v1.WorldRenderEvents;
 import net.fabricmc.fabric.impl.client.rendering.WorldRenderContextImpl;
 
 @Mixin(WorldRenderer.class)
 public abstract class MixinWorldRenderer {
+	@Unique private final WorldRenderContextImpl context = new WorldRenderContextImpl();
```

Comment:
Why move this around? No big deal, but I'd avoid this (seemingly?) unnecessary change.